src/App.tsx

```diff
@@ -21,11 +39,12 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            onChange={(event) => handleChangePerPage(event)}
           >
-            <option value="3">3</option>
-            <option value="5">5</option>
-            <option value="10">10</option>
-            <option value="20">20</option>
+            <option value={3}>3</option>
+            <option selected value={5}>5</option>
```

Comment:
```suggestion
            <option value={5}>5</option>
```
instead selected just pass value={perPage} for select